requirements.json

```diff
@@ -22,7 +22,7 @@
   "cygwin.exe": {
     "bundle": "yes",
     "url": "https://cygwin.com/setup-x86_64.exe",
-    "sha256sum": "08079a13888b74f6466def307a687e02cb26fc257ea2fa78d40f02e28330fd56"
+    "sha256sum": "58f9f42f5dbd52c5e3ecd24e537603ee8897ea15176b7acdc34afcef83e5c19a"
```

Comment:
The new sum seems to be correct... there was something wrong with my configuration